lib/Github/Api/CurrentUser.php

```diff
@@ -104,7 +104,6 @@ public function teams()
      * @param  string $type      role in the repository
      * @param  string $sort      sort by
      * @param  string $direction direction of sort, ask or desc
-
```

Comment:
Well, I would fix this dodgie blank line in the expected way.

Then, fixing the separation of params and return values could be done in a separate PR to make it easier for maintainers to review it, or it could be included in the same PR to have everything fixed at once. I don't know what @pilot prefers